src/macaron/output_reporter/templates/base_template.html

```diff
@@ -683,6 +690,13 @@
                 document.getElementById("check_report_content").classList.toggle("hidden");
             });
 
+            // Add a listener toggle the display of run checks.
```

Comment:
```suggestion
           // Add a listener to toggle the display of run checks.
```

Reply:
Fixed in https://github.com/oracle/macaron/pull/254/commits/7c94792b6406387beec45d59fcbc2002323e8491